esp/src/eclwatch/stub.js

```diff
@@ -30,12 +30,13 @@ define([
     ESPUtil, Utility, LockDialogWidget,
     entities, Toaster) {
 
-        var IDLE_TIMEOUT = cookie("ESPSessionTimeoutSeconds") * 1000;
+        var espTimeoutSeconds = cookie("ESPSessionTimeoutSeconds") || 600;  // 10 Minutes?
```

Comment:
Changing the variable name is fine but no `||` should be allowed. This is a completely configured time and we do not want to limit the user to anything but the configured time. The administrator should make the decision of the configured time not the front end code.

Reply:
When the cached page is loaded, there are no cookies and espTimeoutSeconds === undefined, which means the idle watcher is never getting set, which in turn means it errors when a doIdle topic is published.

Reply:
If it is set then the `|| 600` never comes into play.